src/elements/Label/Label.js

```diff
@@ -136,13 +135,13 @@ Label.propTypes = {
   /** Adds the link style when present, called with (event, props). */
   onClick: PropTypes.func,
 
-  /** Click callback for detail, called with (event, props). */
-  onClickDetail: PropTypes.func,
+  /** Click callback for detail, called with (event, props). Formats the detail as a link. */
+  onDetailClick: PropTypes.func,
 
   /** Adds an "x" icon, called with (event, props) when "x" is clicked. */
-  onClickRemove: PropTypes.func,
+  onRemove: PropTypes.func,
```

Comment:
Updated callbacks and click handlers to make more sense and be consistent with our others (`on*Click`)